.github/workflows/ci.yml

```diff
@@ -3,6 +3,9 @@ on: [push, pull_request]
 env:
   FORCE_COLOR: "1"
   PRE_COMMIT_COLOR: "always"
+  # See https://github.com/theacodes/nox/issues/545
+  # and https://github.com/pre-commit/pre-commit/issues/2178#issuecomment-1002163763
+  SETUPTOOLS_USE_DISTUTILS: "stdlib"
```

Comment:
You might want to do this within nox to avoid this issue locally too, not just in the CI :thinking:

Reply:
Maybe providing an `env={SETUPTOOLS_USE_DISTUTILS: "stdlib"}` to the `session.run`?

Reply:
@gaborbernat I thought that but I've been trying to recreate this issue locally and it works fine, even with the versions of virtualenv and setuptools we're discussing, which is a bit weird!

Reply:
Scratch that! Just cleaned out pre-commit's cache and managed to recreate locally. Good suggestion 👍🏻